packages/cascara/src/modules/DataSelect/DataSelect.js

```diff
@@ -51,11 +51,17 @@ const DataSelect = ({
         ref={formMethods?.register}
       >
         {options ? (
-          options.map((option) => (
-            <option key={option.value} {...option}>
-              {option.label || option.value}
-            </option>
-          ))
+          options.map((option) =>
+            option.value === value ? (
+              <option aria-selected key={option.value} {...option} selected>
```

Comment:
Was eslint telling us to add `aria-selected` here? Just curious. If that is implied by the `selected` prop it would be safe to leave off. I don't think we need to change it, I just want to know if the eslint rule helped us here.